scss/tech.scss

```diff
@@ -1,11 +1,15 @@
-.margin-05 {
-  margin: .5rem 0;
+.main {
+  margin: 0 auto;
+  width: 1360px;
```

Comment:
round numbers, i think using a padding in the container is better. Also, you dont need to have just 3 techs in a row, your design will be ok as long as it looks ok :D